### PR TITLE
12 rsc for models

### LINKED ISSUE #12: RSC for models

### PR DESCRIPTION
**Pull request by Steev-vhoo**
completed the CRUD for achievement, education, experience & license

### CODE REVIEW

File: controllers/achievements.js
Comment (Racheal777) [inline controllers/achievements.js:1]: check your get achievements code very well, try to understand why you are doing what you are doing

Comment (Racheal777) [inline controllers/achievements.js:1]: The update function, why is there value and req.body again, also find all the users experiences before finding one to update, this applies to all the models

Comment (Racheal777) [inline controllers/achievements.js:1]: why are you validating the req.params.id in the delete function

**Review by Racheal777 — COMMENTED** [review]
the comment on the achievement applies to all the other models so check and fix them

File: controllers/achievements.js
Comment (Steev-vhoo) [reply controllers/achievements.js:1]: I would amend for further review

**Pull request closed without merging**